docs/apis/python-bindings/api/htcondor.rst

```diff
@@ -42,6 +42,31 @@ Interacting with Schedulers
    .. automethod:: submit
    .. automethod:: submitMany
    .. automethod:: spool
+
+   A fully-worked example follows.
+
+   .. code-block:: python
+
+        import htcondor
+
+        coll = htcondor.Collector("cm.pool.tld")
+        test = htcondor.classad.quote("remote-schedd.pool.tld")
```

Comment:
Do we want to call this temp variable `test` (also referred later in collector query constraint)... I don't care enough to make you change this but what wanted to point it out

Reply:
Renamed to `quoted_schedd_name` for lack of anything better.